### ISSUE TITLE
THANK YOU

### ISSUE THREAD

**Issue by DreamITSoftware** [CSSEGISandData/COVID-19#2206]
When news of COVID-19 came out I wanted to put together a website with deep analytics.  A few weeks later and we are already getting a lot of traffic to https://www.impulze.live.  I wanted to take the time to thank the team at Johns Hopkins for putting this dataset together #togetherwearebetter

**Comment by judepayne** [comment]
Hi, great site! Many of us here have also put together similar efforts. Understanding helps us get through this tough time. My thanks also to JHU.

**Comment by DreamITSoftware** [comment]
> Hi, great site! Many of us here have also put together similar efforts. Understanding helps us get through this tough time. My thanks also to JHU.

Thank you so much Jude!  Do you have a site as well?  #togetherwearebetter

**Comment by judepayne** [comment]
Yes! https://covid-charts.s3.eu-west-2.amazonaws.com/index.html Something I put together (in clojurescript) to focus on areas of concern for friends and family basically. There's a couple of charts at the bottom - the infection rate and log vs log chart - where I've drawn inspiration from elsewhere and might be of interest.

**Comment by DreamITSoftware** [comment]
> Yes! https://covid-charts.s3.eu-west-2.amazonaws.com/index.html Something I put together (in clojurescript) to focus on areas of concern for friends and family basically. There's a couple of charts at the bottom - the infection rate and log vs log chart - where I've drawn inspiration from elsewhere and might be of interest.

LOVE IT!  You did a great job on this!

**Comment by DreamITSoftware** [comment]
> Yes! https://covid-charts.s3.eu-west-2.amazonaws.com/index.html Something I put together (in clojurescript) to focus on areas of concern for friends and family basically. There's a couple of charts at the bottom - the infection rate and log vs log chart - where I've drawn inspiration from elsewhere and might be of interest.

I would love to connect https://www.linkedin.com/in/danielbhall/
